doc/classes/HeightMapShape3D.xml

```diff
@@ -22,6 +35,16 @@
 				Returns the smallest height value found in [member map_data]. Recalculates only when [member map_data] changes.
 			</description>
 		</method>
+		<method name="update_map_data_from_image">
+			<return type="void" />
+			<param index="0" name="image" type="Image" />
+			<param index="1" name="height_min" type="float" />
+			<param index="2" name="height_max" type="float" />
+			<description>
+				Updates [member map_data] with data read from an [Image] reference in [constant Image.FORMAT_RF] format. Automatically resize heightmap [member map_width] and [member map_depth] to fit the full image width and height.
```

Comment:
Should Image.FORMAT_RH be supported as well? This could help reduce loading times and memory usage for large heightmaps if you don't need as much precision (e.g. if your terrain is relativel flat).

Reply:
Well in that case if we are going for memory and load times might as well also add FORMAT_R8.